db/migrate/20220618102159_create_users.rb

```diff
@@ -4,8 +4,8 @@ def change
         t.string :name
         t.string :photo
         t.text :bio
+        t.integer :posts_counter, default:0
         t.integer :posts_counter
```

Comment:
- [ ] you have duplicated column `posts_counter` in this migration which leads to an error when migrating to a database so please try to remove one and leave only the first one in order to migrate without a problem